CNKI.js

```diff
@@ -232,14 +306,134 @@ function scrape(ids, doc, url, itemInfo) {
 			}
 			
 			i++;
+			
+			// CN 中国刊物编号，非refworks中的callNumber
+			if (newItem.callNumber){
+				newItem.extra = 'CN ' + newItem.callNumber;
+				newItem.callNumber = "";
```

Comment:
Add an (English) explanation as a comment why you moving this information. Moreover, I am not sure it is worth to save this journal identification number at all. Can you say a little bit more, why this could be useful for someone?